test/address/address-lookup.spec.js

```diff
@@ -2,136 +2,65 @@ const chai = require('chai');
 const expect = chai.expect;
 const mockery = require('mockery');
 const nock = require('nock');
-const sinon = require('sinon');
 
 /* The line below turns off ESLints 'no-undef' for the chai 'fail' function */
 /*global fail */
 describe('Address Lookup', () => {
 
-  let addressLookup, extractAddressStub;
+  let addressLookup;
 
   before(function(){
     mockery.enable({
       warnOnReplace: false,
       warnOnUnregistered: false,
       useCleanCache: true
     });
-
-    extractAddressStub = sinon.stub();
-    mockery.registerMock('./extract-address', extractAddressStub);
     addressLookup = require('../../app/address/address-lookup');
-
   });
 
-    it('Should return promise containing addresses extracted from Postcode Service response sorted by field AddressLine1', (done) => {
-
-      let postcode = 'P5TCDE';
-
-      let postcodeServiceResponseAddress1 = {address:'Address1'};
-      let postcodeServiceResponseAddress2 = {address:'Address2'};
-      let postcodeServiceResponseAddress3 = {address:'Address3'};
-      let postcodeServiceResponse = [postcodeServiceResponseAddress1, postcodeServiceResponseAddress2, postcodeServiceResponseAddress3];
-      nock('https://postcodeinfo.service.justice.gov.uk')
-        .get('/addresses?postcode=' + postcode)
-        .reply(200, postcodeServiceResponse);
-
-      let extractedAddress1 = {AddressLine1: '123 Something street'};
-      let extractedAddress2 = {AddressLine1: '1 Something street'};
-      let extractedAddress3 = {AddressLine1: '12 Something street'};
-
-      extractAddressStub.withArgs(postcodeServiceResponseAddress1).returns(extractedAddress1);
-      extractAddressStub.withArgs(postcodeServiceResponseAddress2).returns(extractedAddress2);
-      extractAddressStub.withArgs(postcodeServiceResponseAddress3).returns(extractedAddress3);
-
-      addressLookup(postcode).then((addresses) => {
-        try {
-          expect(addresses.length).to.equal(3);
-          expect(addresses[0]).to.equal(extractedAddress2);
-          expect(addresses[1]).to.equal(extractedAddress3);
-          expect(addresses[2]).to.equal(extractedAddress1);
-          done();
-        }
-        catch (e) {
-          done(e);
-        }
-      });
-   });
+  it('Should expect key to replaced and not the postcode', (done) => {
```

Comment:
key to `be` replaced

Reply:
In this test the key will be replaced from default config value 'AAA' and in the non-prod and prod environment value will come through ccd-vault.